### PR TITLE
add note about new conda-forge packages

### PR DESCRIPTION
**Pull request by dougalsutherland**
I wasn't sure where this should go in the list, so I put it at the top. :p  Feel free to move lower if you think it should be lower.

### CODE REVIEW

File: doc/readme/INSTALL.md [hunk 1]
```diff
@@ -27,6 +28,13 @@ For other cases, we describe how to build Shogun from source code.
 
 ## Ready-to-install packages <a name="binaries"></a>
 
+### Anaconda packages <a name="anaconda"></a>
+The base shogun library and its Python interface are available through the conda package manager, via <a href="https://conda-forge.org">conda-forge</a>. To install both:
```
Comment (karlnapf) [inline doc/readme/INSTALL.md]: Minor: We try to use single lines per sentence

**Comment by karlnapf** [comment]
Fine from my side

**Comment by karlnapf** [comment]
Actually, why don't you also add a thing to NEWS?

**Comment by dougalsutherland** [comment]
@karlnapf Sure....where do I put it? Between 6.0.0 and 6.1.0?

**Comment by karlnapf** [comment]
Just after the 6.1 line, like the cloud/windows/c++11 announcement back for 6.0

**Comment by vigsterkr** [comment]
@dougalsutherland this is like *the* best thing since.... EVER! :) thank you sooooo much for your efforts!

**Pull request merged**